src/applications/static-pages/static-pages-entry.js

```diff
@@ -135,6 +137,7 @@ createCoronavirusChatbot(store, widgetTypes.CORONAVIRUS_CHATBOT);
 createHomepageBanner(store, widgetTypes.HOMEPAGE_BANNER);
 
 createViewDependentsCTA(store, widgetTypes.VIEW_DEPENDENTS_CTA);
+create686CTA(store, widgetTypes.FORM_686_CTA);
```

Comment:
"CTA" generally refers to the sign-in alert boxes like on `/change-direct-deposit/` so you don't have to use that term if there's another you'd prefer.

Reply:
Totally agree, I changed this to reference 'content reveal' in all placed instead of 'CTA'